src/test/java/teammates/test/pageobjects/Browser.java

```diff
@@ -90,7 +91,22 @@ private WebDriver createWebDriver() {
                 System.out.println("Custom path: " + firefoxPath);
                 System.setProperty("webdriver.firefox.bin",firefoxPath);
             }
-            return new FirefoxDriver();
+
+            // Allow CSV files to be download automatically, without a download popup.
+            // This method is used because Selenium cannot directly interact
+            // with the download dialog.
+            // Reference:
+            // http://stackoverflow.com/questions/24852709
+            FirefoxProfile profile = new FirefoxProfile();
+            profile.setPreference("browser.download.panel.shown", false);
+            profile.setPreference("browser.helperApps.neverAsk.openFile",
+                                  "text/csv,application/vnd.ms-excel");
+            profile.setPreference("browser.helperApps.neverAsk.saveToDisk",
+                                  "text/csv,application/vnd.ms-excel");
+            profile.setPreference("browser.download.folderList", 2);
+            profile.setPreference("browser.download.dir",
+                                  System.getProperty("java.io.tmpdir"));
```

Comment:
The previous one was better in terms of readability. Even though the line length limit might be slightly exceeded, it's much easier to comprehend when all the preferences are stacked this way:
```java
profile.setPreference("some.long.preference.1", "value");
profile.setPreference("some.long.preference.2", "some/very/very/very/long/value");
profile.setPreference("some.long.preference.3", value);
```